device/bgp/evpn/bgpv6evpnbroadcastdomainvxlan.yaml

```diff
@@ -0,0 +1,18 @@
+components:
```

Comment:
The contents of this file are identical to that of bgpv4evpnbroadcastdomainvxlan.yaml.  Why do we have separate models for v4 and v6 if the contents are the same?

Reply:
Please check comments below